src/CHANGES.txt

```diff
@@ -6,6 +6,9 @@
 
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
+  From Daniel Moody:
+    - Add no_progress option as a set-able option
```

Comment:
add in parens (-Q) to the description of the change?

Reply:
done